### ISSUE TITLE
[Minor issue] Splash screen logo is pixelated

### ISSUE THREAD

**Issue by EDM115** [ATLauncher/ATLauncher#773]
### What happened?

1. Open ATLauncher
2. Take a look at the splash screen
3. Aaaaah, it's low-quality (didn't knew I started Cyberpunk 2077)

### Operating System

Windows

### Additional Information

I know it's a very minor issue, tho it just doesn't feels right to have that small quality splash screen  
Is it by design to remind the blocky Minecraft style ? 😅  
Furthermore, the [actual splash screen](https://github.com/ATLauncher/ATLauncher/blob/master/src/main/resources/assets/image/splash-screen.png) is crystal clear

![yeah that's a bad quality](https://github.com/ATLauncher/ATLauncher/assets/82015596/6c40e0a5-519b-4fa7-9378-c3362268f037)

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Doomsdayrs** [comment]
[This](https://github.com/ATLauncher/ATLauncher/blob/master/src/main/resources/assets/image/splash-screen.png) is the file. It is only 576x192px.

@RyanTheAllmighty Can we swap these out with SVGs?

**Comment by RyanTheAllmighty** [comment]
I don't believe I have the raw of that. I need to go back through my history with Nodecraft to find it, they provided them back in 2015 to me, and a quick search doesn't bring anything up, so it's entirely possible I never got a raw, but I'll have a look
